server/routes/userRoutes.js

```diff
@@ -15,19 +17,26 @@ async function addStudent(req, res) {
 	res.status(200).json(newStudent);
 }
 
-const loginRouter = () => {
+const loginRouter = async(req, res) => {
```

Comment:
we will have problems with error handling we will need controllers for every route